schema/resolvers.js

```diff
@@ -115,7 +115,16 @@ const Resolvers = {
 
       return answer;
     },
-    updateAnswer: (_, args, ctx) => ctx.repositories.Answer.update(args.input),
+    updateAnswer: (_, args, ctx) => {
+      if (args.input.id.endsWith("to")) {
```

Comment:
I wonder if this kind of business logic should be pushed down into the Answer repository?

Reply:
Yes, I agree. We should try and keep this module as thin as possible.

Reply:
Why are we doing a string check against "to"? Doesn't seem future proof.  What about asserting if its a certain type of answer?

Reply:
Ah I think we are forced into it here as well, the front end sends the same object for both child answers, the only difference between them being the id string.